hbase_pipeline/src/main/java/com/splicemachine/derby/hbase/SpliceIndexEndpoint.java

```diff
@@ -198,15 +199,26 @@ private boolean useToken(BulkWrites bulkWrites) {
         return true;
     }
 
+    // todo: improve this
+    private boolean isLoadReplaceMode(BulkWrites bulkWrites) {
+        boolean loadReplaceMode = false;
+        Iterator<BulkWrite> iterator = bulkWrites.getBulkWrites().iterator();
+        if (iterator.hasNext()) {
+            BulkWrite bw = iterator.next();
+            loadReplaceMode = bw.isLoadReplaceMode();
+        }
```

Comment:
I guess you want to have something similar to reverse iterator, right? (not that it exists AFAIK, but maybe you can wrap this in a special method to make the intention clear.